File: PictsManager/app/src/main/java/com/example/pictsmanager/data/remote/ImageDto.kt
package com.example.pictsmanager.data.remote

import android.graphics.Bitmap
import java.util.Base64
import java.util.Date

data class ImageDto(
    val id: String,
    val label: String,
    val createdAt: Date,
    val extension: String?,
    var base64: String?,
    var compressedBase64: String?,
    @Transient var decodedBitmap: Bitmap?
)

File: Api/Dtos/Image/ImageDto.cs
using System;
using System.Security.Claims;

namespace Api.Dtos
{
	public class ImageDto
	{
        public string Id { get; set; } = string.Empty;
        public string Label { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public string Extension { get; set; } = string.Empty;
        public string Base64 { get; set; } = string.Empty;
        public string CompressedBase64 { get; set; } = string.Empty;

        public ImageDto() { }

    }
}

